spk/beets/src/requirements.txt

```diff
@@ -1,29 +1,29 @@
 # Produced by pip freeze
 # Commented if provided by python package at /usr/local/python/share/wheelhouse/
-beets==1.4.3
-click==6.7
+beets==1.4.7
+click==7.0
 # For Discogs
 discogs-client==2.2.1
 #enum34==1.1.6
 # For Web Plugin
-Flask==0.12
-Flask-Cors==3.0.2
+Flask==1.0.2
+Flask-Cors==3.0.6
 itsdangerous==0.24
-jellyfish==0.5.6
-Jinja2==2.9.5
+jellyfish==0.6.1
+Jinja2==2.10
 #MarkupSafe==0.23
-munkres==1.0.8
+munkres==1.0.11
```

Comment:
I used `pip-upgrade` to upgrade all the dependencies

however build failed for `munkres==1.0.12` so I set it to `1.0.11`

output error
```
Collecting munkres==1.0.12 (from -r /mnt/d/user/workspace/spksrc/spk/beets/work/wheelhouse/requirements.txt (line 15))
  Could not find a version that satisfies the requirement munkres==1.0.12 (from -r /mnt/d/user/workspace/spksrc/spk/beets/work/wheelhouse/requirements.txt (line 15)) (from versions: 1.0.5.4, 1.0.6, 1.0.7, 1.0.8, 1.0.10, 1.0.11)
No matching distribution found for munkres==1.0.12 (from -r /mnt/d/user/workspace/spksrc/spk/beets/work/wheelhouse/requirements.txt (line 15))
make[1]: *** [build_wheel_target] Error 1
```